ajax/libs/scion/package.json

```diff
@@ -1,17 +1,27 @@
 {
   "name": "scion",
-  "filename": "scion-min.js",
-  "version": "0.0.3",
+  "version": "3.0.0",
   "description": "An implementation of SCXML in JavaScript.",
   "keywords": [
     "scxml",
     "statecharts",
     "w3c",
     "javascript"
   ],
+  "author" : "Jacob Beard <jake@jacobeanrnd.com>",
```

Comment:
Because of the format check, there cannot be space before colon mark. Let's modify it to `"author":` and `filename` field also. Or you can run `tools/fixFormat.js` to help you fix it. Thanks.